examples/gocean/eg1/compute_cu_mod.f90

```diff
@@ -16,9 +16,9 @@ module compute_cu_mod
 
   type, extends(kernel_type) :: compute_cu
      type(arg), dimension(3) :: meta_args =    &
-          (/ go_arg(GO_WRITE, GO_CU, GO_POINTWISE),        & ! cu
-             go_arg(GO_READ,  GO_CT, GO_POINTWISE),        & ! p
-             go_arg(GO_READ,  GO_CU, GO_POINTWISE)         & ! u
+          (/ go_arg(GO_WRITE, GO_CU, GO_POINTWISE),            & ! cu
+             go_arg(GO_READ,  GO_CT, GO_STENCIL(000,011,000)), & ! p
```

Comment:
This kernel accesses P(i,j) and P(i-1, j) so I think this should be: (000,110,000). In case you haven't seen it, the documentation is at: https://psyclone.readthedocs.io/en/stable/gocean1p0.html#metadata

Also, it might be clearer if you break the lines, e.g.:

    GO_STENCIL(000, &
               110, &
               000)

Reply:
Oops yes, I misinterpreted one of the directions and then done all of them wrong, fixed now.